### PR TITLE
Check that libc linux headers are present in host before sysbox builds.

### PR DESCRIPTION
**Pull request by ctalledo**
Signed-off-by: ctalledo <ctalledo@nestybox.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -299,6 +299,10 @@ test-cleanup: test-img
 	$(DOCKER_RUN) /bin/bash -c "testContainerCleanup"
 	$(TEST_DIR)/scr/testContainerPost $(TEST_VOL1) $(TEST_VOL2) $(TEST_VOL3)
 
+host-prereq:
+	@printf "\n** Checking host prerequisites **\n\n"
+	@test -f /usr/include/linux/seccomp.h || \
+		(echo 'Your host does not have the /usr/include/linux/seccomp.h header file; please install it (e.g., sudo apt-get install linux-libc-dev)'; exit 1)
```
Comment (rodnymolina) [inline Makefile:305]: Don't we have to look into this file to verify that it meets our internal seccomp requirements? [i'm thinking about PR #78].

Comment (ctalledo) [reply Makefile:305]: Yes and no. 

We need a seccomp.h that has the seccomp notify definitions (e.g., SECCOMP_IOCTL_NOTIF_RECV, SECCOMP_IOCTL_NOTIF_ID_VALID, etc.) 

But we don't need a seccomp.h that has the latest definition of SECCOMP_IOCTL_NOTIF_ID_VALID; rather we need a seccomp.h whose definition of this symbol matches the one in the kernel (as described in issue #78).

I think we should add a check to verify that the seccomp notify definitions are present. Let me rework this.

Comment (rodnymolina) [reply Makefile:305]: Yep, that's what i meant.

**Review by rodnymolina — CHANGES_REQUESTED** [review]
See comments.

**Comment by ctalledo** [comment]
After investigating, we noticed this PR is incorrect. I'll create a fresh new one. Closing.

**Pull request closed without merging**